src/openrct2/core/File.cpp

```diff
@@ -24,9 +24,9 @@
 #include "File.h"
 #include "FileStream.hpp"
 #include "String.hpp"
+#include "../util/Util.h"
 
 #include "../platform/platform.h"
-#include "../util/Util.h"
```

Comment:
Why this mode?

Reply:
@Broxzier sorry, I did not see your comment until after I merged. Somehow github does not refresh properly today.

That's still a remnant of the `extern "C" {}` times, I think, where the includes had to be split. I must have initially removed this include and manually add it later. Would be solved with https://github.com/OpenRCT2/OpenRCT2/pull/7178